src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +26,13 @@
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<? super Machine> machines, Object value);
```

Comment:
```suggestion
    void fill(List<? super T> machines, T value);
```
краще використати T, це дасть більше гнучкості при оголошенні імплементації. Так типу буде динамічно братись для цим методів замість хардоку Object чи Machine